jekyll/_cci2/local-cli.md

```diff
@@ -24,6 +24,8 @@ You can also run `circleci` commands in your `config.yml` file for jobs that use
 
 ### Ubuntu 16.04+, Fedora 27+, and other Distros Supporting Snap Packages
 
+With `snapd` installed (pre-installed on Ubuntu), the Local CLI can be installed on various Linux distros. This gives you the CircleCI Local CLI, Docker, and the security & auto-update features that come with [Snap packages](https://www.ubuntu.com/desktop/snappy).
```

Comment:
Appreciate the context! I'd adjust to:

"After installing `snapd`, you can install the Local CLI on various Linux distributions. Installing the Local CLI also gives you Docker, as well as [Snap packages](https://www.ubuntu.com/desktop/snappy)' security and update features."

Reply:
I took some of this. Thanks.